java/cut-geometry/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
What you've done here is very clean, but for the sake of presenting a common experience across samples can you make the MapView cover the full screen and the button float over top, like in the other samples.

We should look at the rest of the samples and see if this solution (button(s) next to the MapView rather than over it works and think about changing all of them to match). However for now, let just keep it looking like the others.